### PR TITLE
Adding Support Policy for Microsoft Support

### PR DESCRIPTION
**Pull request by uday31in**
**This PR fixes**

Adding Support Policy for Microsoft Support

### CODE REVIEW

File: SUPPORT.md [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
+- Underlying RP issues when deploying template (e.g. MGs, Policies LA, vWAN, vNet) for any deployment failure
+- Subscription Creation via portal experience
+- UI elements (e.g., ArmApiController) used in ARM Template
+- RP Issues e.g. template validation, Checkaccess API etc.
+- Customization of Portal Template/Policy and UI Definition by customer/partners
 
-Support for this **PROJECT or PRODUCT** is limited to the resources listed above.
+Any issues that deemed outside of the above list by Microsoft support and/or requires bugfix in the Template or Code in the repo, Microsoft support will redirect user to file the issue on GitHub.
+
+Project maintainers and community aim to get issues resolved in timely fashion as per community support policy of this repo.
```
Comment (alex-frankel) [inline SUPPORT.md:18]: Issue may take more than 3 days to resolve. For GitHub repos that are supported, our rule is "72 hours to get a meaningful response from the product team".

File: SUPPORT.md [hunk 2]
```diff
@@ -1,13 +1,28 @@
...
-This project uses GitHub Issues to track bugs and feature requests. Please search the existing 
-issues before filing new issues to avoid duplicates.  For new issues, file your bug or 
-feature request as a new Issue.
+This repo is supported by Microsoft Support and users will be able to engage Microsoft support via their usual channels. Please provide corelation IDs where possible when contacting support to be able to investigate issue effectively and in timely fashion.
 
-For help and questions about using this project, please submit a Github issue with corresponding [Issue Labels found here](https://github.com/Azure/Enterprise-Scale/labels). Responses will be posted within 24 Hours. 
+Following list of issues are considered within realm when contacting Microsoft support:
 
-## Microsoft Support Policy  
+- Portal deployment of 4 Reference implementation - WingTip, Contoso and AdventureWorks, TreyResearch
```
[the remark this answers is not in the document]
Comment (uday31in) [reply SUPPORT.md:9]: It is clarified at the end of the paragraph

Any issues that deemed outside of the above list by Microsoft support **and/or requires bugfix in the Template or Code in the repo,** Microsoft support will redirect user to file the issue on GitHub.

File: SUPPORT.md [hunk 3]
```diff
@@ -1,13 +1,28 @@
 # Support
 
-## How to file issues and get help  
+## Microsoft Support Policy
 
-This project uses GitHub Issues to track bugs and feature requests. Please search the existing 
-issues before filing new issues to avoid duplicates.  For new issues, file your bug or 
-feature request as a new Issue.
+This repo is supported by Microsoft Support and users will be able to engage Microsoft support via their usual channels. Please provide corelation IDs where possible when contacting support to be able to investigate issue effectively and in timely fashion.
```
[the remark this answers is not in the document]
Comment (uday31in) [reply SUPPORT.md]: Added the link https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/deployment-history?tabs=azure-portal#get-deployments-and-correlation-id

File: SUPPORT.md [hunk 4]
```diff
@@ -1,13 +1,28 @@
...
-This project uses GitHub Issues to track bugs and feature requests. Please search the existing 
-issues before filing new issues to avoid duplicates.  For new issues, file your bug or 
-feature request as a new Issue.
+This repo is supported by Microsoft Support and users will be able to engage Microsoft support via their usual channels. Please provide corelation IDs where possible when contacting support to be able to investigate issue effectively and in timely fashion. For instruction on how to get deployments and correlation ID, please follow this link [here](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/deployment-history?tabs=azure-portal#get-deployments-and-correlation-id).
 
-For help and questions about using this project, please submit a Github issue with corresponding [Issue Labels found here](https://github.com/Azure/Enterprise-Scale/labels). Responses will be posted within 24 Hours. 
+Following list of issues are within the scope of Microsoft support:
 
-## Microsoft Support Policy  
+- Portal deployment of 4 Reference implementation - WingTip, Contoso and AdventureWorks, TreyResearch
```
Comment (krnese) [inline SUPPORT.md:9]: Let's remove Trey Research as it's not ready yet (readme is being worked on by FT)

File: SUPPORT.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (rkuehfus) [reply SUPPORT.md]: Any reason why we use "aim" vs. "commit" on this statement?

Project maintainers will aim to respond within 3 business days to get a meaningful response for any new issues.

[the remark this answers is not in the document]
Comment (uday31in) [reply SUPPORT.md]: It isn't SLA and hence it is SLO.

[the remark this answers is not in the document]
Comment (alex-frankel) [reply SUPPORT.md]: Yes looks good

File: SUPPORT.md [hunk 5]
```diff
@@ -1,13 +1,28 @@
...
-This project uses GitHub Issues to track bugs and feature requests. Please search the existing 
-issues before filing new issues to avoid duplicates.  For new issues, file your bug or 
-feature request as a new Issue.
+This repo is supported by Microsoft Support and users will be able to engage Microsoft support via their usual channels. Please provide corelation IDs where possible when contacting support to be able to investigate issue effectively and in timely fashion. For instruction on how to get deployments and correlation ID, please follow this link [here](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/deployment-history?tabs=azure-portal#get-deployments-and-correlation-id).
 
-For help and questions about using this project, please submit a Github issue with corresponding [Issue Labels found here](https://github.com/Azure/Enterprise-Scale/labels). Responses will be posted within 24 Hours. 
+Following list of issues are within the scope of Microsoft support:
 
-## Microsoft Support Policy  
+- Portal deployment of 4 Reference implementation - WingTip, Contoso and AdventureWorks
```
Comment (alex-frankel) [inline SUPPORT.md:9]: This says 4 still, but now there are 3

**Review by alex-frankel — DISMISSED** [review]

**Review by krnese — APPROVED** [review]

**Review by kwill-MSFT — COMMENTED** [review]
The opening statement "This repo is supported by Microsoft Support and users will be able to ..." is misleading.  The repo isn't supported, but customers deploying Azure resources is supported.   I would change it to "If issues are encountered when deploying these reference implementations users will be able to ..."

**Review by kwill-MSFT — APPROVED** [review]

**Review by krnese — APPROVED** [review]

**Pull request merged**